iidm/iidm-serde/src/test/java/com/powsybl/iidm/serde/extensions/ActivePowerControlXmlTest.java

```diff
@@ -50,4 +52,51 @@ void test() throws IOException {
         assertEquals(activePowerControl.getParticipationFactor(), activePowerControl2.getParticipationFactor(), 0.0);
         assertEquals(activePowerControl.getName(), activePowerControl2.getName());
     }
+
+    @Test
+    void testIidmV10() throws IOException {
+        Network network = BatteryNetworkFactory.create();
+        Battery bat = network.getBattery("BAT");
+        assertNotNull(bat);
+
+        ActivePowerControl<Battery> activePowerControl = new ActivePowerControlImpl<>(bat, true, 4.0, 1.2);
+        bat.addExtension(ActivePowerControl.class, activePowerControl);
+
+        Generator generator = network.getGenerator("GEN");
+        generator.addExtension(ActivePowerControl.class, new ActivePowerControlImpl<>(generator, false, 3.0, 1.0));
+
+        Network network2 = allFormatsRoundTripTest(network, "/batteryNetworkWithActivePowerControlRoundTripRef.xml", IidmVersion.V_1_0);
+
+        Battery bat2 = network2.getBattery("BAT");
+        assertNotNull(bat2);
+        ActivePowerControl<Battery> activePowerControl2 = bat2.getExtension(ActivePowerControl.class);
+        assertNotNull(activePowerControl2);
+
+        assertEquals(activePowerControl.isParticipate(), activePowerControl2.isParticipate());
+        assertEquals(activePowerControl.getDroop(), activePowerControl2.getDroop(), 0.0);
+        assertEquals(activePowerControl.getParticipationFactor(), activePowerControl2.getParticipationFactor(), 0.0);
+        assertEquals(activePowerControl.getName(), activePowerControl2.getName());
+    }
+
+    @Test
+    void testIidmV10WithoutActivePowerControlExtension() throws IOException {
```

Comment:
I think there's no use to test this, as this feature of excluding at export one or all extension is already tested in `ExportOptionsTest`.